### ISSUE TITLE
Leaderboard improvements

### ISSUE THREAD

**Issue by seeReadCode** [coyote-team/coyote#42]
Other leaderboard component ideas:
- non-progress bar-able important metrics, such as ones that hopefully count from n to 0, e.g. the ready to review count,
- comparably show counts for top 5 describers as well as v. self,
- show badges, names for some of the bests, e.g. approval record, days active streak.

We would need agreement on the important metrics, and then a graphic treatment per metric type.  Some of these ideas would require much more dev time than others to implement.

(originated in #32)

**Comment by seeReadCode** [comment]
@alavatelliMCA @TobeyeboT it would really help to unblock progress on this if we could have some related design work by end of week.  Please let me know if you have questions.

**Comment by alavatelliMCA** [comment]
Sorry Chris, the in-line updates to the other issue confused me because I wasn't sure what changed. 

Whichever way we go, we do want to keep it simple and straightforward. These were the metrics we found most important included in the design document, plus I added the top describers idea:

Our Status
 # Described Images
 # Total Descriptions
 # Approved Descriptions
 # Pending Descriptions Ready for Review
 # Unapproved Descriptions
 # Undescribed Images with Open Assignments (admin only)
 # Images Unassigned (admin only)

Top Describers:
 # Name, # Name, # Name

My Status
 # Described Images
 # Undescribed Images Assigned to You
 # Approved Descriptions
 # Pending Descriptions
 # Unapproved Descriptions

total number and % to be displayed

I am sorry to let this one languish I didn't realize there was a pitch to redesign. I'll look at these in with Tobey in context of what he had proposed (page 5 of the design document).

@sinabahram let me know your thoughts on scope and the stats.

**Comment by seeReadCode** [comment]
@alavatelliMCA thanks!  

The main design issue is that many of these make sense as numbers rather than progress bars.  I can move forward with generating a table of these and we can work on styling later as need be.

**Comment by sinabahram** [comment]
I like this approach. It gets the data there with minimal effort, and exposes it in a reasonable way. Visual treatment can happen as design and more budget becomes available.

**Comment by alavatelliMCA** [comment]
A note: In grand scheme of work to be done this is lower priority than the remaining work on list views, it doesn't seem the design has been applied to these yet. Let me know if there is anything else you need.

As for this issue: I am not thrilled about an undesigned table leading the page when we have a good design. It is useful to have the simple progress bar to give the numbers a context, otherwise they are pretty abstract. 

I added info in parenthesis next to each statistic to identify what the percentage is of to demonstrate the usefulness. Each stat leads with a straight number figure and then has the progress bars below. Page 5 of the design sketch would still be the basis. I like top describers, but not necessary for this phase if we want to simplify this work.

**Our Status**
# Total Descriptions

# Described Images (% of Images)
# Approved Descriptions (% of Descriptions)

updated admin only list
# Pending Descriptions Ready for Review (admin only) (% of Descriptions)
# Unapproved Descriptions (admin only) (% of Descriptions)
# Undescribed Images with Open Assignments (admin only)  (% of Images)
# Images Unassigned (admin only)  (% of total Images)

**My Status**
 # Undescribed Images Assigned to You

 # Images Described by You (% of Described Images) - note label change
 # Approved Descriptions (% of Your Descriptions)
 # Pending Descriptions (% of Your Descriptions)
 # Unapproved Descriptions (% of Your Descriptions)

total number and % to be displayed

**Comment by seeReadCode** [comment]
Re: #44 it might make sense to feature the latest image timestamp as pulled from MCA.

**Comment by seeReadCode** [comment]
I implemented these and moved the leaderboard below the queues.

**Comment by alavatelliMCA** [comment]
I don't understand why it was moved to the bottom of the page, perhaps there is something I am not seeing? The idea to add it was to create incentive to people, being able to see our progress and one's own.  Seeing a zero when I logged in was sad and I immediately wanted to get describing! :)

If it feels too top heavy we can edit the selection of stats down to a shorter list.

**Comment by seeReadCode** [comment]
@alavatelliMCA I did that only because of your comment about not wanting an unstyled table at the top of the page.  I'm glad to move it back up.  Any other changes you'd like for it?

**Comment by alavatelliMCA** [comment]
Ah, I see what we currently have as being styled as it matches our design. I thought you had proposed something even more stripped down. Simple misunderstanding!

Functional Question: Seems like everyone is zero on their own stats, if we can't make those work we can scrap it, but it is nice to see personal progress.

For legibility, I would love a comma:
12,422

Here is my list modified with editorial notes (case changes) and user feedback. This should clarify text and order of items and phrasing of statistics to be more clear

Our Status

# Described images of _total#_ images (% of images) 
# Images assigned and undescribed of _total#_ images (% of images) _(admin only)_ 
# Images unassigned of _total#_ images (% of total images)  _(admin only)_
# Approved descriptions of _total#_ descriptions (% of descriptions) 
# Descriptions ready for review  of _total#_ descriptions (% of descriptions) _(admin only)_ 

REMOVE: # Unapproved descriptions _(admin only)_ (% of descriptions) of total# 

My Status

# Images described of _total#_ images described  (% of described Images) 
# Approved descriptions of your descriptions (% of your descriptions) 
# Unapproved descriptions of your descriptions (% of your descriptions) 

REMOVE: # Pending descriptions of your descriptions (% of Your Descriptions) of your descriptions


Thank you so much and sorry to be fussy with this one!
